### DISCUSSION TITLE
Bottom sheet only for mobile devices?

### DISCUSSION THREAD

**Discussion by vytautas-pranskunas-** [taiga-family/taiga-ui#9575] [General]
Hi, i see the sentence that bottom sheet only works on mobile devices just here: https://taiga-ui.dev/components/sheet-dialog

I am using it to select customers from grid and it works normal on normal full width browser. Does that mean that i should stop using it and look for other solutions or?

**Comment by waterplea** [comment]
During recent refactors I believe I dropped all the mobile specific code, so I'll remove this sentence. It might just look a bit odd when you scroll it on desktop, otherwise everything should work.
